src/components/Chat/ChatInput/ChatInput.tsx

```diff
@@ -160,13 +167,30 @@ export const ChatInput = ({
       return;
     }
 
-    onSend({ role: 'user', content });
+    onSend({
+      role: 'user',
+      content,
+      ...(files.length > 0 && {
+        custom_content: {
+          attachments: files
+            .filter(
+              (file) => file.status !== 'FAILED' && file.status !== 'UPLOADING',
+            )
+            .map((file) => ({
+              type: file.contentType,
+              title: file.name,
+              url: encodeURI(file.absolutePath + '/' + file.name),
+            })),
+        },
+      }),
```

Comment:
this block isn't readable. As I understood correctly it should be like this:
```suggestion
      custom_content: files.length > 0 ? {
          attachments: files
            .filter(
              (file) => file.status !== 'FAILED' && file.status !== 'UPLOADING',
            )
            .map((file) => ({
              type: file.contentType,
              title: file.name,
              url: encodeURI(`${file.absolutePath}/${file.name}`),
            })),
        },
      } : undefined,
```
but even better to move this code into util-method `getCustomContent(files)` outside this component/file and use:
``` javascript
custom_content: getCustomContent(files)
```